docs/reference/api/rest.md

```diff
@@ -1,9 +1,15 @@
 ---
-description: Web3Signer REST API reference
+description: Use for signing consensus layer payloads
 sidebar_position: 2
 ---
 
-# REST API
+# Web3Signer REST API
+
+The Web3Signer REST API contains an ETH2 (that is, consensus layer) API, and an ETH1 (that is, execution layer) API. 
+Use the ETH2 API for signing consensus layer payloads. 
+
+We recommend using the [Web3Signer JSON-RPC API](json-rpc.md) for signing execution layer payloads. The ETH1 REST API contains a basic signing method but does not implement transaction encoding
```

Comment:
Doesn't this contradict what's on line 2? I guess I'd keep the description on line number 2 more generic.

Reply:
I don't think so. 

Line 2 says to use the REST API for consensus layer payloads. 

Line 11 says the REST API can be used for signing execution layer payloads but we recommend using the JSON-RPC for execution layer signing.  Line 2 doesn't say anything about execution layer payloads.  That's by design because if they're looking for something to sign execution layer payloads, they don't even need to click into the REST API. 

Does that make sense?  Or I'm missing something?

Reply:
It makes sense